lib/src/importer/filesystem.dart

```diff
@@ -20,6 +20,8 @@ class FilesystemImporter extends Importer {
   final String _loadPath;
 
   /// Creates an importer that loads files relative to [loadPath].
+  /// @nodoc
+  @internal
```

Comment:
@nex3 Is the change you requested in our meeting? My recollection was that this was attempting to prevent many `FilesystemImporters` from being created, which doesn't match my understanding of this annotation. I think I'm misunderstanding either the problem or change, and want to double check.

Reply:
No, marking the existing constructor as internal and nodoc is effectively a breaking change for anyone who's already using it :sweat_smile:. I think what I suggested is creating a top-level `final` `FilesystemImporter('.')` so that it can be shared among multiple users, and marking _that_ as internal and nodoc.

Reply:
@nex3 To make sure I understand-

- Add an `internal/nodoc` FilesystemImporter to `lib/sass.dart`
- Everywhere there is a `FilesystemImporter('.')` created, replace that by importing the singleton from `import 'package:sass/sass.dart' as sass;`

I'm seeing roughly 15 of these created throughout `dart-sass`, so I want to confirm before I make the change.

Reply:
Changed in c22c8ca32c0df0660da66c9c0c740b3aead195d2